scss/atoms/typography/_typography.scss

```diff
@@ -5,3 +5,120 @@
 @import
 	"variables",
 	"functions";
+
+// TODO probably should do the import in the `head` in case character sets need to change dynamically
+@import url( http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700,400italic,700italic );
```

Comment:
This results in an `@import` in the final min file and is against the style guide. This should just be an additional link in the `.html` file.